### PR TITLE
*: Cut v2.6.0

### PR DESCRIPTION
**Pull request by mrueg**
Planned ChangeLog:

## v2.6.0 / 2022-08-26
* [FEATURE]     Add local storage labels to kube_persistentvolume_info #1814 @nabokihms
* [FEATURE]     Add support for StateSet and Info metrics for Custom-Resource State #1777 @chrischdi
* [FEATURE]     Add support for Rolebinding resource metrics #1799 @kaitoii11
* [FEATURE]     Add new kube_horizontalpodautoscaler_status_target_metric #1725 @tanguyfalconnet
* [FEATURE]     Add support for Role and Clusterrole resource metrics #1759 @kaitoii11
* [FEATURE]     Add support for the ServiceAccount resource metrics #1773 @Serializator
* [FEATURE]     Add metric for pod tolerations #1769 @Serializator
* [FEATURE]     Add new kube_endpoint_address metric #1761 @bavarianbidi
* [FEATURE]     Support reading the timestamp fields of custom resources #1766 @dontan001
* [ENHANCEMENT] Migrate to structured logging #1807 @dmpe
* [ENHANCEMENT] Switch registry vanity domain from k8s.gcr.io to registry.k8s.io #1750 @mrueg
* [ENHANCEMENT] Graduate new endpoint metrics to stable #1812 @bavarianbidi
* [BUGFIX]      Fix label name for kube_pod_nodeselector #1756 @yosshi825
* [BUGFIX]      Fix Custom-Resource State Metrics not to contain underscores #1754 @chrischdi
* [BUGFIX]      Fix kube_node_status_allocatable unit doc #1760 @jumbosushi

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,19 @@
...
+* [FEATURE]     Add support for Role and Clusterrole resource metrics #1759 @kaitoii11
+* [FEATURE]     Add support for the ServiceAccount resource metrics #1773 @Serializator
+* [FEATURE]     Add metric for pod tolerations #1769 @Serializator
+* [FEATURE]     Add new kube_endpoint_address metric #1761 @bavarianbidi
+* [FEATURE]     Support reading the timestamp fields of custom resources #1766 @dontan001
+* [ENHANCEMENT] Switch registry vanity domain from k8s.gcr.io to registry.k8s.io #1750 @mrueg
+* [ENHANCEMENT] Graduate new endpoint metrics to stable #1812 @bavarianbidi
+* [BUGFIX]      Fix label name for kube_pod_nodeselector #1756 @yosshi825
+* [BUGFIX]      Fix Custom-Resource State Metrics not to contain underscores #1754 @chrischdi
+* [BUGFIX]      Fix ube_node_status_allocatable unit doc #1760 @jumbosushi
```
Comment (bavarianbidi) [inline CHANGELOG.md:15]:
```suggestion
* [BUGFIX]      Fix kube_node_status_allocatable unit doc #1760 @jumbosushi
```

Comment (mrueg) [reply CHANGELOG.md]: Good catch, thanks!

**Comment by dgrisonnet** [comment]
https://github.com/kubernetes/kube-state-metrics/pull/1814 is merged, are there any PRs that we are still waiting for?

**Comment by mrueg** [comment]
> #1814 is merged, are there any PRs that we are still waiting for?

I've included it, there are no other PRs that I would want in 2.6.0 as it's going to be a pretty big release.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,19 @@
+## v2.6.0 / 2022-08-26
+* [FEATURE]     Migrate to structured logging #1807 @dmpe
```
[the remark this answers is not in the document]
Comment (mrueg) [reply CHANGELOG.md]: Thanks, moved it to enhancement!

**Pull request merged**